### PR TITLE
add clarinet@0.11.0 w/ npm auto-update

### LINKED ISSUE #9761: [Request] Add clarinet.js
**Library name:** Clarinet.js
**Git repository url:** https://github.com/dscape/clarinet
**npm package url(optional):** https://www.npmjs.com/package/clarinet
**License(s):** BSD
**Official homepage:** https://github.com/dscape/clarinet
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster than just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by qwe19272375**
Pull request for issue: #9761 
Related issue(s): #9414 
@nolsherry please help me review this PR, thanks!

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/dscape/clarinet
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/clarinet
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 318
   - Count of watchers: 18
   - Count of forks: 192
 * NPM download stats (optional):
   - Downloads in the last day: 860
   - Downloads in the last week: 5719
   - Downloads in the last month: 21412

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #9761, cc @dscape #9414

### CODE REVIEW

File: ajax/libs/clarinet/package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  "npmName": "clarinet",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "clarinet*"
+      ]
+    }
+  ],
+  "license": "BSD"
```
Comment (nolsherry) [inline ajax/libs/clarinet/package.json]: @qwe19272375 Please modify `BSD` to `BSD-2-Clause`.

**Comment by nolsherry** [comment]
LGTM, @cdnjs/intern2

**Comment by qwe19272375** [comment]
@kennynaoh map file added. Please help me review it, thanks!

File: ajax/libs/clarinet/package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "clarinet",
+  "filename": "clarinet.min.js",
+  "description": "SAX based evented streaming JSON parser in JavaScript (browser and node)",
+  "author": "Nuno Job <nunojobpinto@gmail.com> (http://nunojob.com/)",
+  "version": "0.11.0",
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/dscape/clarinet.git"
```
Comment (kennynaoh) [inline ajax/libs/clarinet/package.json]: @qwe19272375 It's better to use `https` instead of `http`. Thanks!

**Comment by nolsherry** [comment]
LGTM, @qwe19272375 please tag me first after you fix the problem next time, thanks!
@kennynaoh please help me review this PR, thanks!

**Comment by kennynaoh** [comment]
@qwe19272375 Please rebase to the latest master branch, thanks!

**Comment by qwe19272375** [comment]
Commit updated.

**Comment by kennynaoh** [comment]
LGTM, ping @PeterDaveHello

**Comment by dscape** [comment]
Do you guys need some input from me?

Many thanks

**Comment by PeterDaveHello** [comment]
@dscape nope, just let you know that we added this lib on CDNJS :)

**Comment by dscape** [comment]
Cool! Thanks

**Pull request merged**